hpx/compute/cuda/value_proxy.hpp

```diff
@@ -68,6 +68,12 @@ namespace hpx { namespace compute { namespace cuda
             return access_target::read(*target_, p_);
         }
 #endif
+
+        T* operator &() const
+        {
+            return p_;
+        }
+
         T* device_ptr() const HPX_NOEXCEPT
```

Comment:
Nope. Even if yes, the signature of `std::adressof() ` (return type being `T*` and argument being `T&`) does not make things easy